### PR TITLE
Weather: Make the embed cleaner by reformatting it

### PR DESCRIPTION
**Pull request by sinus-x**
- Data presentation has been changed (no more bullet points)
- Today's day phases in the past are not being displayed
- "Server" is used instead of "Guild" for user-targeted messages
- Command "weather-place" has been added to display the defaults

### CODE REVIEW

**Review by ArcasCZ — COMMENTED** [review]
Looks good to me

File: weather/module.py [hunk 1]
```diff
@@ -41,6 +42,16 @@ def _translate_day_phase(self, ctx: commands.Context, day_phase: str) -> str:
         if day_phase == "Night":
             return _(ctx, "Night")
 
+    def _get_current_day_phase(self, ctx: commands.Context) -> str:
+        now = datetime.datetime.now()
+        if now.hour <= 6:
+            return _(ctx, "Morning")
```
[the remark this answers is not in the document]
Comment (ArcasCZ) [reply weather/module.py:48]: Never mind, didn't notice that this translation in popie must match API and it's not just for displaying information purposes.

**Review by BorekZnovustvoritel — APPROVED** [review]
Did not find anything weird in the code or by testing.

**Review by czechbol — APPROVED** [review]
Looks perfectly fine to me

**Pull request merged**